src/sys/wait.rs

```diff
@@ -145,27 +145,34 @@ mod status {
           target_os = "netbsd"))]
 mod status {
     use sys::signal::Signal;
+    use libc::c_int;
```

Comment:
I don't understand why you use ```i32``` here instead of ```c_int```. The latter is the correct type for those APIs, as waitpid returns a c_int, not a i32. On most platforms these types are identical, but c_int should be the correct one.